apps/client/src/pages/WrapperPage.js

```diff
@@ -35,36 +72,82 @@ const WrapperPage = ({ checker, topic, resetChecker }) => {
             highlightActive
           >
             <StepByStepItem
-              active={!finishedLocation}
-              checked={finishedLocation}
+              active={isActive("locationInput") || isActive("locationResult")}
+              checked={isFinished("locationResult")}
               heading="Adres gegevens"
               largeCircle
             >
-              <Location resetChecker={resetChecker} checker={checker} />
+              {(!activeComponents || isActive("locationInput")) && (
```

Comment:
You didn't find a way to init an activeComponent so we can drop the `!activeComponents` ?